kubernetes/README.md

```diff
@@ -726,4 +726,60 @@ This is what the query on `envoy_http_downstream_cx_tx_bytes_total` will return:
 
 ### 30. Visualize mesh with Kuma GUI
 
-Kuma ships with an internal GUI that will help you visualize the mesh and its policies in an intuitive format. It can be found on port `:5683` on the control-plane pod. We port-forwarded this port [earlier](#10-now-we-will-port-forward-the-kuma-control-plane-so-we-can-access-it-with-kumactl) so now we can access the GUI at [http://localhost:5683/](http://localhost:5683/).
\ No newline at end of file
+Kuma ships with an internal GUI that will help you visualize the mesh and its policies in an intuitive format. It can be found on port `:5683` on the control-plane pod. We port-forwarded this port [earlier](#10-now-we-will-port-forward-the-kuma-control-plane-so-we-can-access-it-with-kumactl) so now we can access the GUI at [http://localhost:5683/](http://localhost:5683/).
+
+### 31. Kong Gateway Integration 
+
+The `Dataplane` can now operate in Gateway mode. This way you can integrate Kuma with existing API Gateways like [Kong](https://github.com/Kong/kong). When you use a Dataplane with a service, both inbound traffic to a service and outbound traffic from the service flows through the Dataplane. This API Gateway should be deployed as any other service within the mesh. Use the `kuma-demo-kong.yaml` file to deploy [Kong for Kubernetes](https://github.com/Kong/kubernetes-ingress-controller):
```

Comment:
This paragraph is too terse for me:
* `Dataplane` in Gateway mode
* existing API Gateways
* This API Gateway

Too many "gateways" without explaining how they are different and what is role of each of them.